### PR TITLE
include global abstain/no option

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
election polls with 'one vote per candidate' should have global no/abstain options

### CODE REVIEW

File: client/src/app/site/assignments/services/assignment-poll.service.ts [hunk 1]
```diff
@@ -83,6 +88,7 @@ export class AssignmentPollService extends PollService {
                     if (Math.min(...yes) < 0) {
                         return null;
                     } else {
+                        // TODO: Counting 'No (and possibly 'Abstain') here?
```
Comment (MaximilianKrambach) [inline client/src/app/site/assignments/services/assignment-poll.service.ts]: this is the discussion point I need: how do 'no to  all' and 'abstain' influence the percent base?

Comment (MaximilianKrambach) [reply client/src/app/site/assignments/services/assignment-poll.service.ts]: in 2.3 client, these values don't affect the calculation. I'm unsure if they should.

Comment (FinnStutzenstein) [reply client/src/app/site/assignments/services/assignment-poll.service.ts]: @emanuelschuetze Ping

Comment (MaximilianKrambach) [reply client/src/app/site/assignments/services/assignment-poll.service.ts]: e clarified in direct discussion: We will continue as in 2.3

**Comment by MaximilianKrambach** [comment]
@emanuelschuetze reviewable now

**Pull request merged**